### ISSUE TITLE
Dark Hydra, Evil Planet Lord - Not Automated

### ISSUE THREAD

**Issue by ZeroVash** [naveed92/dm-ocg-octgn#10]
This one is a tricky one to automate, albeit not hard.

Having a dictionary for ContinuousOnSummon Effects could provide useful.

**Comment by naveed92** [comment]
Yes, this would solve Mist Rias, Moritz and Thrumyss as well (among others)

**Comment by ZeroVash** [comment]
I'll push my updates on the weekend then :)
